src/fillTank.test.js

```diff
@@ -1,11 +1,111 @@
-'use strict';
+"use strict";
 
-describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+describe("fillTank", () => {
+  const { fillTank } = require("./fillTank");
 
-  it('should ', () => {
+  it('should be declared', () => {
```

Comment:
Actually we don't need this test: if `fillTank` is not a function, any other test would fail with the corresponding error.

Reply:
Deleted this test)